src/workerd/api/wpt/url-test.ts

```diff
@@ -0,0 +1,144 @@
+      'request.formData() with input: ',
+      'response.formData() with input: ',
+      'request.formData() with input: a',
+      'response.formData() with input: a',
+      'request.formData() with input: a=b',
+      'response.formData() with input: a=b',
+      'request.formData() with input: a=',
+      'response.formData() with input: a=',
+      'request.formData() with input: =b',
+      'response.formData() with input: =b',
+      'request.formData() with input: &',
+      'response.formData() with input: &',
+      'request.formData() with input: &a',
+      'response.formData() with input: &a',
+      'request.formData() with input: a&',
+      'response.formData() with input: a&',
+      'request.formData() with input: a&a',
+      'response.formData() with input: a&a',
+      'request.formData() with input: a&b&c',
+      'response.formData() with input: a&b&c',
+      'request.formData() with input: a=b&c=d',
+      'response.formData() with input: a=b&c=d',
+      'request.formData() with input: a=b&c=d&',
+      'response.formData() with input: a=b&c=d&',
+      'request.formData() with input: &&&a=b&&&&c=d&',
+      'response.formData() with input: &&&a=b&&&&c=d&',
+      'request.formData() with input: a=a&a=b&a=c',
+      'response.formData() with input: a=a&a=b&a=c',
+      'request.formData() with input: a==a',
+      'response.formData() with input: a==a',
+      'request.formData() with input: a=a+b+c+d',
+      'response.formData() with input: a=a+b+c+d',
+      'request.formData() with input: %=a',
+      'response.formData() with input: %=a',
+      'request.formData() with input: %a=a',
+      'response.formData() with input: %a=a',
+      'request.formData() with input: %a_=a',
+      'response.formData() with input: %a_=a',
+      'request.formData() with input: %61=a',
+      'response.formData() with input: %61=a',
+      'request.formData() with input: %61+%4d%4D=',
+      'response.formData() with input: %61+%4d%4D=',
+      'request.formData() with input: id=0&value=%',
+      'response.formData() with input: id=0&value=%',
+      'request.formData() with input: b=%2sf%2a',
+      'response.formData() with input: b=%2sf%2a',
+      'request.formData() with input: b=%2%2af%2a',
+      'response.formData() with input: b=%2%2af%2a',
+      'request.formData() with input: b=%%2a',
+      'response.formData() with input: b=%%2a',
+    ],
+  },
+  'urlsearchparams-constructor.any.js': {
+    comment: 'Fix this eventually',
+    expectedFailures: [
+      'URLSearchParams constructor, DOMException as argument',
+      'Construct with 2 unpaired surrogates (no trailing)',
+      'Construct with 3 unpaired surrogates (no leading)',
+      'Construct with object with NULL, non-ASCII, and surrogate keys',
+    ],
+  },
+  'urlsearchparams-sort.any.js': {
+    comment: 'Figure out why we sort differently than the WPT tests',
+    expectedFailures: ['Parse and sort: ﬃ&🌈', 'URL parse and sort: ﬃ&🌈'],
```

Comment:
This is probably related to how sort() function is implemented in Ada https://github.com/ada-url/ada/blob/main/include/ada/url_search_params-inl.h#L165

Good opportunity to contribute to Ada